functions.js

```diff
@@ -0,0 +1,50 @@
+const ourTest = {
+    capitalize: (str) => {
+        return `${str[0].toUpperCase()}${str.substr(1, str.length-1)}`;
+    },
+    reverse: (str) => {
+        return str.split("").reverse().join("");
+    },
+    calculator: {
+        add: (x, y) => x + y,
+        subtract: (x, y) => x - y,
+        divide: (x, y) => x / y,
+        multiply: (x, y) => x * y,
+    },
+    caesar: (str, num) => {
+        const letters = [
+            'a', 'b', 'c', 'd', 'e', 'f',
+            'g', 'h', 'i', 'j', 'k', 'l',
+            'm', 'n', 'o', 'p', 'q', 'r',
+            's', 't', 'u', 'v', 'w', 'x',
+            'y', 'z'
+        ];
+        let output = "";
+        str.split("").forEach(letter => {
+            const letterIndex = letters.indexOf(letter);
+            if ((letterIndex + num) > 25) {
+                output += letters[(letterIndex + num) % 25];
+            } else {
+                output += letters[letterIndex + num];
+            }
```

Comment:
we still need to test for the following;
1. same case
2. punctuation!
![image](https://user-images.githubusercontent.com/31897434/85197060-8f96ed80-b2d6-11ea-985f-0f414d03137e.png)